### ISSUE TITLE
"First Tracker" column doesn't show anything

### ISSUE THREAD

**Issue by xlucn** [transmission-remote-gtk/transmission-remote-gtk#218]
The transmission version is 4.0.0. So probably it's due to trg not compatible with the new version?

**Comment by LaserEyess** [comment]
Yes, there are new RPC calls and information that trg hasn't been updated to support. Some of it is easy, some of it requires serious refactoring (like labels).
